LogcatCoreLib/src/main/java/info/hannes/logcat/LogcatFragment.kt

```diff
@@ -58,6 +58,5 @@ class LogcatFragment : LogBaseFragment() {
             fragment.arguments = args
             return fragment
         }
-
```

Comment:
Sometimes remove a line makes only noise and should be separated in a format commit.
I'm not speaking about necessary of it.